lib/member_name_parts.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberNameParts < Scraped::HTML
+  field :prefix do
+    partitioned.first.join(' ')
+  end
+
+  field :name do
+    # .sub called for specific case where Dr. is not separated from name by space
+    partitioned.last.join(' ').sub('Dr.Bharrat', 'Bharrat')
```

Comment:
Again, this makes the output differ from the output before the scraper was refactored as the scraper was not capturing Dr Bharrat's professional title.

Do you think I should make the change in this PR?